pygotham/talks/models.py

```diff
@@ -107,3 +108,30 @@ def is_accepted(self):
     def slug(self):
         """Return a slug for the instance."""
         return slugify(self.name, max_length=25)
+
+
+class Speaker(db.Model):
+
+    """Speaker, represents the intent to give a talk"""
+
+    __tablename__ = 'speakers'
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    talk_id = db.Column(
+        db.Integer, db.ForeignKey('talks.id'), nullable=False,
+    )
+    talk = db.relationship(
+        'Talk', backref=db.backref('talks', lazy='dynamic'),
+    )
+
+    user_id = db.Column(
+        db.Integer, db.ForeignKey('users.id'), nullable=False
+    )
+    user = db.relationship(
+        'User', backref=db.backref('users', lazy='dynamic')
```

Comment:
`secondary` is probably the correct behavior here. I think we'll most often reference things like `user.talks` and `talk.speakers`, so this makes things significantly less verbose.